### PR TITLE
Add method GetMessage to interface OpenFlowModMessage

### PR DESCRIPTION
**Pull request by hongliangl**
Add method `GetMessage` to interface `OpenFlowModMessage` to get the member `message` of the struct which implements the interface `OpenFlowModMessage`.

Signed-off-by: Hongliang Liu <lhongliang@vmware.com>

### CODE REVIEW

File: ofctrl/fgraphFlow.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
-/***
+/*
```
Comment (wenyingd) [inline ofctrl/fgraphFlow.go:1]: No need for this change?

**Review by wenyingd — COMMENTED** [review]
Overall LGTM, only one minor comment

File: ofctrl/fgraphFlow.go (hunk 1 above)
Comment (hongliangl) [reply ofctrl/fgraphFlow.go:1]: It is formatted by Goland automatically.

**Review by wenyingd — APPROVED** [review]
LGTM

**Comment by wenyingd** [comment]
@tnqn Do you have other comments on this PR? If not, I would merge it.

**Review by tnqn — APPROVED** [review]
LGTM

**Pull request merged**